src/util/docCookies.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Helper module use to get, set and remove item from cookie
+ *
+ * See more:
+ *  https://developer.mozilla.org/en-US/docs/Web/API/document.cookie
+ *  https://developer.mozilla.org/User:fusionchess
+ *  https://github.com/madmurphy/cookies.js
+ *  This framework is released under the GNU Public License, version 3 or later.
+ *  http://www.gnu.org/licenses/gpl-3.0-standalone.html
+ */
+
+/**
+ * Get specific key's value stored in cookie
+ *
+ * @param {string} sKey
+ *
+ * @returns {string|null}
+ */
+export function getItem(sKey) {
+  if (!sKey) return null;
+
+  return decodeURIComponent(
+    // eslint-disable-next-line no-useless-escape
+    document.cookie.replace(new RegExp('(?:(?:^|.*;)\\s*' + encodeURIComponent(sKey).replace(/[\-\.\+\*]/g, '\\$&') + '\\s*\\=\\s*([^;]*).*$)|^.*$'), '$1')) || null;
+}
+
+/**
+ * Set specific key's value in cookie
+ *
+ * @param {string} sKey cookie name
+ * @param {string} sValue cookie value
+ * @param {string} vEnd expire|max-age
+ * @param {string} sPath path of current item
+ * @param {string} sDomain domain name
+ * @param {boolean} bSecure true|false
```

Comment:
This is how you denote optional parameters in jsdoc:

```suggestion
 * @param {string} sKey cookie name
 * @param {string} sValue cookie value
 * @param {string} [vEnd] expire|max-age
 * @param {string} [sPath] path of current item
 * @param {string} [sDomain] domain name
 * @param {boolean} [bSecure]
```

Reply:
I am not sure how vEnd, sPath, sDomain are optional parameter. why we are donating them using [] braces. @cdrini !

Reply:
Hey! So these are optional because they don't need to be specified; e.g. calling `setItem('test-cookie', 'jack-sparrow')` (like we do in the tests) is a normal way to call this function.